Source/WebCore/html/HTMLFrameElement.cpp

```diff
@@ -56,7 +58,8 @@ bool HTMLFrameElement::rendererIsNeeded(const RenderStyle& style)
 
 RenderPtr<RenderElement> HTMLFrameElement::createElementRenderer(RenderStyle&& style, const RenderTreePosition&)
 {
-    return createRenderer<RenderFrame>(*this, WTFMove(style));
+    CheckedRef renderStyle = style;
+    return createRenderer<RenderFrame>(*this, WTFMove(renderStyle.get()));
```

Comment:
The guideline for this case is to protect at the call site, not use protectors.

Reply:
The pre-existing code was totally safe here, this was a false positive. Ryosuke is aware and is working on fixing it. In the mean time, we should silence the warning, not try and protect with a local variable.